kale/pipeline/video_domain_adapter.py

```diff
@@ -625,39 +659,24 @@ def critic_update_steps(self, batch):
                 h_s = torch.cat((h_s_rgb, h_s_flow), dim=1)
                 h_t = torch.cat((h_t_rgb, h_t_flow), dim=1)
 
+            # Need to improve to process rgb and flow dividedly in the future.
             for _ in range(self._k_critic):
-                gp = losses.gradient_penalty(self.domain_classifier, h_s, h_t)
-
-                critic_s = self.domain_classifier(h_s)
-                critic_t = self.domain_classifier(h_t)
-                wasserstein_distance = critic_s.mean() - (1 + self._beta_ratio) * critic_t.mean()
-
-                critic_cost = -wasserstein_distance + self._gamma * gp
-
-                self.critic_opt.zero_grad()
-                critic_cost.backward()
-                self.critic_opt.step()
-                if self.critic_sched:
-                    self.critic_sched.step()
-
-                # Uncomment for later work. Process rgb and flow dividedly.
-                # for _ in range(self._k_critic):
-                #     gp_rgb = losses.gradient_penalty(self.domain_classifier, h_s_rgb, h_t_rgb)
-                #     gp_flow = losses.gradient_penalty(self.domain_classifier, h_s_flow, h_t_flow)
-                #
-                #     critic_s_rgb = self.domain_classifier(h_s_rgb)
-                #     critic_s_flow = self.domain_classifier(h_s_flow)
-                #     critic_t_rgb = self.domain_classifier(h_t_rgb)
-                #     critic_t_flow = self.domain_classifier(h_t_flow)
-                #     wasserstein_distance_rgb = (
-                #             critic_s_rgb.mean() - (1 + self._beta_ratio) * critic_t_rgb.mean()
-                #     )
-                #     wasserstein_distance_flow = (
-                #             critic_s_flow.mean() - (1 + self._beta_ratio) * critic_t_flow.mean()
-                #     )
-                #
-                #     critic_cost = (-wasserstein_distance_rgb + -wasserstein_distance_flow +
-                #                    self._gamma * gp_rgb + self._gamma * gp_flow) * 0.5
+                gp_rgb = losses.gradient_penalty(self.domain_classifier, h_s_rgb, h_t_rgb)
```

Comment:
what's `gp`? Gaussian process? `grad_pen(alty)`? Some abbreviations are hard to understand or remember.